core/src/main/java/org/apache/cloudstack/storage/to/VolumeObjectTO.java

```diff
@@ -357,4 +366,22 @@ public String getvSphereStoragePolicyId() {
     public void setvSphereStoragePolicyId(String vSphereStoragePolicyId) {
         this.vSphereStoragePolicyId = vSphereStoragePolicyId;
     }
+
+    public String getEncryptFormat() { return encryptFormat; }
+
+    public void setEncryptFormat(String encryptFormat) { this.encryptFormat = encryptFormat; }
+
+    public byte[] getPassphrase() { return passphrase; }
+
+    public void setPassphrase(byte[] passphrase) { this.passphrase = passphrase; }
```

Comment:
Cool, I'll review those and change them. We should probably add an exception for single line blocks, seems cleaner, but that's an opinion to discuss another day :-) Will get on this.

Reply:
@JoaoJandre - I was going to change these, but in reviewing VolumeObjectTO, if I change them they'll be (almost) the only getters and setters that are not single line blocks, and there are a lot of them. What do you think, is it better to adhere to the existing style, and perhaps come back and audit the code if the standard dictates changes are needed - or do we just have mixed style as new commits come in?

Reply:
Reading through the code conventions, there is emphasis on "should" versus "must", and that readability comes before convention. I'm willing to change the style here, but absent any further feedback, I would opt to maintain the existing convention of this class and leave it as-is rather than mixing conventions.